scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -82,8 +82,20 @@
       <div>
         <strong>Local (arch.)</strong>: {{ variant.local_obs_old or 0 }} obs.
       </div>
+      {% if variant.cosmic_ids %}
+      <strong>CosmicIDs</strong>:
+      <div>
+        {% for cosmicID in variant.cosmic_ids %}
```

Comment:
This might be a bit too long list. Some INDEL might overlap with >3 COSMIC SNVs. It is a challenge with how VEP works.

Could it be useful if we only show count? I know we talked about showing the ID, but now I see that it is not informative.

Reply:
example: `COSMIC obs.: 1`

Reply:
Done!

Is now, "COSMIC: 2 obs". Order chosen to match "Local (arch)" on the line above.